Frontend/voitheia-App/src/app/components/home/home.component.html

```diff
@@ -1,15 +1,41 @@
-<div *ngIf="isLoggedIn()">
+<div class="container h-100 content-container">
+    <div style="margin-left: 10%; margin-top: 4.5rem; margin-bottom: 4rem;" class="row h-100 ">
+        <div class="col-lg-10 align-self-end">
+            <h1 class="text-white font-weight-bold">Help yourself<br>by helping others</h1>
+        </div>
+        <div class="col-lg-10 align-self-baseline">
+            <p class=" font-weight-light mb-5">
+                {{'Home.Texts.Level1' | translate }}
+                <br>
+                {{'Home.Texts.Level2' | translate }}
+            </p>
+            <div class="">
+                <a style="float:left" (click)="_navigationService.navigateTo('request/create')"
+                    class="btn btn-primary btn-xl">{{'Home.Buttons.CreateRequest' | translate }}</a>
```

Comment:
We should change the buttons to Material Buttons.

Reply:
changed

Reply:
also changed navigationservice to route and removed public navservice